py3status/modules/khal_calendar.py

```diff
@@ -38,6 +39,7 @@ class Py3status:
     date_end = "eod"
     format = "{appointments}"
     output_format = "{start-time} {title}"
+    limit = None
```

Comment:
Configs not in alphabetical order.

Reply:
thanks, I have addressed your comment in 9e89f9d